packages/integrations/useJwt/index.ts

```diff
@@ -0,0 +1,28 @@
+import jwt_decode, { JwtDecodeOptions, JwtHeader, JwtPayload } from 'jwt-decode'
+import { computed, Ref } from 'vue-demi'
+
+/**
+ * Reactive decoced jwt token.
+ *
+ * @see {@link https://vueuse.js.org/integrations/useJwt/}
+ * @param jwt
+ */
+export function useJwt<Payload extends object = JwtPayload, Header extends object = JwtHeader>(encodedJwt: Ref<string>) {
+  const decodeWithFallback = <T extends object>(encodedJwt: string, options?: JwtDecodeOptions) => {
+    try {
+      return jwt_decode<T>(encodedJwt, options)
+    }
+    catch (err) {
```

Comment:
Not sure about this, is the error doesn't matter, or should we let users to capture this error if need?

Reply:
We could also return an the error beside payload and header.

Reply:
I would keep the fallback and maybe add an optional callback function, which enables users to react on errors. IMO an additional `error` return param does not make sense, since checking this param requires more effort than passing a simple callback. What do you think?

Reply:
Sounds good :)